R/ml_kmeans.R

```diff
@@ -54,19 +56,33 @@ ml_kmeans <- function(x,
   # extract cluster centers
   kmmCenters <- invoke(fit, "clusterCenters")
 
+  # compute cost for k-means
+  if (compute.cost)
```

Comment:
I have really no idea. 

This is this method
https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeansModel.scala#L84-L88 

I think you have to subtract from each observation it's corresponding cluster center. The take square of this operation, and take sum for every operation for all observations. This looks trivial and can be easily done parallel. I haven't noticed a big difference in computing time when clustering 100 columns and 7 mln observations.

Should we go compute.cost = TRUE?